Lib/test/test_codecs.py

```diff
@@ -1641,6 +1641,13 @@ def test_register(self):
         self.assertRaises(TypeError, codecs.register)
         self.assertRaises(TypeError, codecs.register, 42)
 
+    def test_unregister(self):
+        search_function = mock.Mock(return_value=(1, 2, 3, 4))
```

Comment:
If you want to use it for real (see my comment below), it should return a valid value: None.

```suggestion
        search_function = mock.Mock(return_value=None)
```

Reply:
Oh, I use `(1, 2, 3, 4)` to test search is called or not in my local vm~, and decide remove those test codes(I thouht this test case forcus on the process of `unregister`). 
I will update it soon.